girder/models/user.py

```diff
@@ -187,10 +187,9 @@ def search(self, text=None, user=None, limit=50, offset=0, sort=None):
         else:
             cursor = self.find({}, limit=0, sort=sort)
 
-        for r in self.filterResultsByPermission(cursor=cursor, user=user,
-                                                level=AccessType.READ,
-                                                limit=limit, offset=offset):
-            yield r
+        return self.filterResultsByPermission(
```

Comment:
In the function documentation, we claim that we return a "List of users", and now we return a cursor with the result set of users.  We probably should change that.  Strangely, this is the only function that was so modified that even has documentation on the return parameter.